src/main/java/racingcar/domain/Judgement.java

```diff
@@ -4,24 +4,25 @@
 import java.util.List;
 
 public class Judgement {
-    public List<String> judgeWinner(List<Car> carList) {
-        int maxPoint;
-        List<String> winners = new ArrayList<>();
 
-        maxPoint = findMaxPoint(carList);
-        for (int i = 0; i < carList.size(); i++) {
-            if (carList.get(i).getProgress() == maxPoint) {
-                winners.add(carList.get(i).getName());
-            }
-        }
-        return winners;
+  public List<String> judgeWinner(List<Car> carLineUp) {
+    int maxPoint;
+    List<String> winners = new ArrayList<>();
+
+    maxPoint = findMaxPoint(carLineUp);
+    for (int i = 0; i < carLineUp.size(); i++) {
+      if (carLineUp.get(i).getProgress() == maxPoint) {
```

Comment:
`carLineUp.get(i).getProgress() == maxPoint` 가 의미하는 것은 해당 자동차가 '우승자인지' 확인하는 것이겠죠? 이렇게 비교하는 구문이 특정한 의미를 가질 때에는 maxPoint 와 같은지 비교하는 구문을 `Car` 의 메서드로 만들어주면 가독성이 더 좋아진답니다! 다음과 같이 고쳐볼 수 있을 것 같아요 🙂

```suggestion
      if (carLineUp.get(i).isWinner(maxPoint)) {
```

Reply:
코드를 작성하고 검토하면서 생각해 볼 만한 점을 하나 더 갖게되었습니다. 😄  
Car 클래스의 메서드로 수정하였습니다.